app/views/nominations/request_nomination_invite/review.html.erb

```diff
@@ -14,6 +14,10 @@
       <%= EmailDecorator::new(@nomination_request_form.school.primary_contact_email).partial_email %>
     </p>
 
+    <p class="govuk-body">
+      We've redacted the email address held in the GIAS register for data security reasons. It may belong to a staff member, or many schools use a shared email address starting with a word like 'admin', 'office', 'info' or 'headteacher'.
```

Comment:
Wonder if we could cut this down a bit? Also I thought we partially redacted for privacy reasons not security reasons? 🤷 

I did wonder if "redacted" is a bit jargony, but I couldn’t think of a plainer alternative...? 🤔 

```suggestion
      We’ve partially redacted the email address for privacy reasons. It may belong to a staff member or be a shared email address.
```

<img width="707" alt="Screenshot 2023-09-19 at 13 24 22" src="https://github.com/DFE-Digital/early-careers-framework/assets/30665/5a631349-2597-429b-9959-ff971d57f2a1">

Reply:
```suggestion
      We've redacted the email address held in the GIAS register for privacy reasons. It may belong to a staff member, or many schools use a shared email address starting with a word like 'admin', 'office', 'info' or 'headteacher'.
```

Reply:
Hi, I've updated the reason. It feels important to include the GIAS information in case users miss it earlier in the journey. Based on support queries, it feels important to include some of the common email prefixes too. We could use bullets instead, e.g.:

We've redacted part of the email address held in the GIAS register for privacy reasons. It may:

- be a shared email address starting with a word like 'admin', 'office', 'info' or 'headteacher'
- belong to a staff member